### ISSUE TITLE
Bootstrap separation from main application

### ISSUE THREAD

**Issue by niccolomineo** [coderedcorp/coderedcms#235]
**Is your feature request related to a problem? Please describe.**
Our Front-Enders realise they would much more prefer to work with a clean Django template, as they're progressively doing without Bootstrap due to the functionality overhead it adds.

**Describe the solution you'd like**
I'd like to see Bootstrap as a separate package / toggleable feature / "how-to" guide in the documentation.

**Comment by vsalvino** [comment]
Many people have asked this. I think at some point we will separate them in the future, perhaps once we are able to support themes. For now though, and at least not until after a solid 1.0 release, we don't plan to separate Bootstrap. That would probably happen for a 2.0 release.

The solution for now, is to either override the HTML templates, or if you are doing a very deeply custom website implementation, then "pure" wagtail might be a better fit.

**Comment by pauloxnet** [comment]
Very interesting plan. Thanks for your works on codered !

**Comment by niccolomineo** [comment]
Hi, any progress on this?

**Comment by vsalvino** [comment]
No progress on this at the moment. Separating bootstrap is a pretty low priority for us so there is no ETA, but any progress that does happen will be posted here on this issue.
